starport/cmd/network_chain_list.go

```diff
@@ -67,11 +59,18 @@ func renderLaunchSummaries(chainLaunches []networktypes.ChainLaunch, out io.Writ
 			campaign = fmt.Sprintf("%d", c.CampaignID)
 		}
 
+		reward := "no reward"
```

Comment:
```suggestion
		reward := "-"
```

This will probably make the cli output more clean. Wdyt?